samples/OpenThread/UdpThreadClient/Program.cs

```diff
@@ -0,0 +1,170 @@
+using System;
+using System.Threading;
+using System.Net;
+using nanoFramework.Networking.Thread;
+
+namespace Samples
+{
+    public class Program
+    {
+        const int UDP_PORT = 1234;
+
+        static OpenThread ot;
```

Comment:
like for the constant, we should be explicit, so "private" and also for global variables, even in samples, we start with underscore so it will be `private static OpenThread _ot;`
Same for the event